portfolio/src/main/webapp/index.html

```diff
@@ -4,39 +4,41 @@
     <meta charset="UTF-8">
     <title>Lindsey's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Chewy">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="introduction">
       <h1>Hey, I'm Lindsey.</h1>
       <h2>It's nice to meet you.</h2>
-      <p>This is me.</p>
+      <h2>This is me.</h2>
       <img src="images/profilepic.JPG"/>
     </div>
-
-    <div id="projects">
+    <p></p>
+    <div id="content">
```

Comment:
Hmm. I noticed you use the same id name for multiple chunks of code. Ids should be unique to single code blocks. For what you're doing, classes are better. see: https://css-tricks.com/the-difference-between-id-and-class/

Reply:
I changed all of the non-unique ids to classes and reflected that in style.css!